pkg/innerring/processors/netmap/process_peers.go

```diff
@@ -37,6 +44,37 @@ func (np *Processor) processAddPeer(ev netmapEvent.AddPeer) {
 		return
 	}
 
+	var info clientcore.NodeInfo
```

Comment:
why don't we extend `np.nodeValidator.VerifyAndUpdate` procedure below with new check?

Reply:
have not thought about that. it does not seem to me like a node info structure validator

Reply:
ok, moved to a separate validator